### DISCUSSION TITLE
有待改进

### DISCUSSION THREAD

**Discussion by NeuronCState** [YaoFANGUK/video-subtitle-remover#33] [General]
我最开始就想找一个能像AI去除照片水印的那种方式，对视频逐帧去水印。好不容易找到了这个。感觉还是训练样本不够多有的时候水印都去不干净能明显看到字。下一个版本可以不大面积蒙版框选字幕水印区域，改为更精细的选区只选择字迹，类似PS抠图一样。这样会不会好一些？

**Comment by YaoFANGUK** [comment]
主要问题是没钱训练

**Reply by lqlnbcr** [reply]
真实了

**Reply by lqlnbcr** [reply]
这个能力的需求还是很大的，愿早日拿到融资

**Comment by SweetSourPeter** [comment]
试用了STTN，效果一般

LAMA和PROPAINTER模式报错：
ExternalError: CUDNN error(9), CUDNN_STATUS_NOT_SUPPORTED. 
      [Hint: Please search for the error code(9) on website (https://docs.nvidia.com/deeplearning/cudnn/api/index.html#cudnnStatus_t) to get Nvidia's official solution and advice about CUDNN Error.] (at C:\home\workspace\Paddle\paddle\fluid\operators\fused\conv_fusion_op.cu:519)
      [operator < conv2d_fusion > error]
Subtitle Finding:   0%|          | 0/805 [0

**Comment by nowkin** [comment]
have the same error even with cuda18

**Comment by 1998-BossLee** [comment]
一分钱难倒好汉
